### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by jmonto55**
This PR contains all the steps followed to build the third part of the [Vet Clinic Database project](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md) , including:

- Creation of a table named owners.
- Creation of a table named species.
- Added new queries.
- Added queries using JOIN

**Screenshots of results**

img2
img1
img3

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -118,3 +118,47 @@ SELECT species AS type_of_animal, AVG(escape_attempts) AS average_of_escape_atte
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY type_of_animal;
+
```
Comment (alexander16108) [inline queries.sql:121]: You are required to add the following data to the data.sql file which is  currently unavailable 
- for the owner's table data
```
 INSERT INTO owners (full_name, age)
 VALUES ('Sam Smith', 34);

 INSERT INTO owners (full_name, age)
 VALUES ('Jennifer Orwell', 19);

 INSERT INTO owners (full_name, age)
 VALUES ('Bob', 45);

 INSERT INTO owners (full_name, age)
 VALUES ('Melody Pond', 77);

 INSERT INTO owners (full_name, age)
 VALUES ('Dean Winchester', 14);

 INSERT INTO owners (full_name, age)
 VALUES ('Jodie Whittaker', 38);
```
- for the species table  data
```
 INSERT INTO species (name)
 VALUES ('Pokemon');

 INSERT INTO species (name)
 VALUES ('Digimon');
```

- `animals` table should be modified to meet these requirements
     - if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead
     - all animals got assigned owners

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @jmonto55 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✅ Correct use of GitHub Flow
✅ Descriptive Pull Request
✅ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (jmonto55) [reply queries.sql:121]: Thank you, I did not notice that my changes were not committed. @alexander16108

**Comment by arslanbisharat** [comment]
Hi @jmonto55,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**